app/views/issues/_toolbar.html.erb

```diff
@@ -0,0 +1,41 @@
+<%#
+  Inspired in the Gmail toolbar. Some of the controls are hidden while nothing
+  is selected and they become visible once the user selects at least one row.
+%>
+<div class="btn-toolbar">
+  <div class="btn-group">
+     <button class="btn">
```

Comment:
This button believes differently from the other buttons in the toolbar, which is unintuitive. For the other buttons, I can click *anywhere* on the button - the icon or the grey background - to make it work. But here, the button itself (i.e. the grey part) doesn't work; I have to click exactly on the checkbox.